test/cross-browser/reference.html

```diff
@@ -22,36 +20,51 @@
 
     <!-- traces -->
     <script>
-        var childWin;
+        var clientLogger;
+
+        function createLocalStorageLogger() {
+            return {
+                debug: function() {
+                    localStorage.setItem(
```

Comment:
My initial thought was, "who cares? why would I ever want to set length, key, setItem, getItem, removeItem, or clear?" It would be horrible practice to do so.

But I buy the argument of using function implementation so that it is apparent it has different behavior than default objects.